src/maintenance/performance.rst

```diff
@@ -212,6 +212,21 @@ This TCP buffering behaviour can be disabled via
 .. seealso::
     Bulk :ref:`load <api/db/all_docs>` and :ref:`store <api/db/bulk_docs>` API.
 
+Connection limit
+----------------
+
+`MochiWeb`_ is handling CouchDB requests.
+By default maximum number of connections is 2048. To change this limit, use
+server_options configuration variable. 'max' indicates maximum number of
```

Comment:
Let's use backticks \` for config string like say \`server_options\` or \`max\`

Reply:
The sentence is also missing an article. It should read: "To change this limit, use the server_options configuration variable."